README.md

```diff
@@ -27,6 +27,7 @@ The current members of the CHIPS Alliance TSC are:
 | Matt Cockrell      | [mcockrell-google](https://github.com/mcockrell-google) | Google                 | RISC-V DV                 | (appointed) | (appointed) |
 | Tom Michalak       | [tmichalak](https://github.com/tmichalak)               | Antmicro               | F4PGA                     | (appointed) | (appointed) |
 | Mikhail Moiseev    | [mikhailmoiseev](https://github.com/mikhailmoiseev)     | Intel                  | SystemC compiler          | (appointed) | (appointed) |
+| Jiuyang Liu        | [sequencer](https://github.com/sequencer)               | unaffiliated           | Rocket Chip               | (appointed) | (appointed) |
```

Comment:
```suggestion
| Jiuyang Liu        | [sequencer](https://github.com/sequencer)               | PLCT Lab, ISCAS        | Rocket Chip               | (appointed) | (appointed) |
```
Thanks for adding this! Since my sponsor is [PLCT Lab](https://plctlab.github.io/), I'm requesting to add it as my affiliation.